src/main/java/com/sda/eventapp/web/rest/controller/EventRestController.java

```diff
@@ -14,18 +14,19 @@
 
 @Slf4j
 @RestController
-@RequestMapping("/event")
+@RequestMapping({"/events"})
```

Comment:
Maybe we will think about REST paths (URLs) in convention like "XXX/api/v1/events"? This would make it clear that these are api calls.
Let me know, @degorskiprzemyslaw, @Adrie0291.